arisia-remote/app/arisia/controllers/ZoomController.scala

```diff
@@ -22,4 +25,32 @@ class ZoomController(
       Ok("Got it!")
     }
   }
+
+  def enterItem(itemStr: String): EssentialAction = Action { implicit request =>
+    val redirectOpt = for {
+      // Only logged in users are allowed to join meetings:
+      user <- LoginController.loggedInUser()
+      attendeeUrl <- scheduleService.getAttendeeUrlFor(user, ProgramItemId(itemStr))
+      // If we get here, they're allowed in:
+    }
+      yield Found(attendeeUrl)
+
+    // TODO: what should we return if this fails? This is effectively a system error: it shouldn't be possible for them
```

Comment:
i think we should try to assume good faith and be as specific as possible about the failure - whether it's that the user wasn't logged in or there's not a valid panel right now or that it's asking about an event that got cancelled and the page hasn't refreshed, etc